.env.example

```diff
@@ -41,3 +41,8 @@ GITHUB_APP_PRIVATE_KEY=""
 
 # GitHub App Name in your App's Installation URL (https://github.com/apps/${GITHUB_APP_SLUG}/installations/new)
 GITHUB_APP_SLUG=
+
+#Trigger API credential  URL(https://trigger.dev/docs/documentation/introduction)
+TRIGGER_API_KEY=
+TRIGGER_API_URL=
+NEXT_PUBLIC_TRIGGER_PUBLIC_API_KEY=
```

Comment:
Do we need the `NEXT_PUBLIC_TRIGGER_PUBLIC_API_KEY`?

Reply:
We really don't need this for now

Reply:
Can we remove this then across the app?